introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla lorem augue, egestas scelerisque imperdiet semper, imperdiet eu diam. Pellentesque vulputate nunc eget est malesuada eleifend. Sed mollis convallis tempor. Nulla sed convallis eros. Proin lectus ante, commodo ac euismod at, dignissim eget orci. Suspendisse molestie magna laoreet tellus condimentum tempus. In adipiscing suscipit lobortis.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.